operator/v1/types_ingress.go

```diff
@@ -501,6 +535,12 @@ type AccessLogging struct {
 	//
 	// +optional
 	HttpLogFormat string `json:"httpLogFormat,omitempty"`
+
+	// httpCaptureHeaders defines HTTP headers that should be captured in
+	// access logs.  If this field is empty, no headers are captured.
```

Comment:
what does this do for routes that are not terminated at the router?

Reply:
Nothing.  We cannot read (or write) the HTTP headers of requests or responses for TLS passthrough routes.

Reply:
I added godoc explaining that the setting does not apply if the connection is TLS passthrough.  I added a similar note to the godoc for `httpLogFormat`.